lte/gateway/docker/docker-compose.yaml

```diff
@@ -20,6 +20,23 @@ x-standard-volumes: &volumes_anchor
   - /var/log:/var/log
   - /etc/openvswitch:/etc/openvswitch
 
+# Magmad volumes mounted
+x-magmad-volumes: &magmad_volumes_anchor
+  - ${CERTS_VOLUME}:/var/opt/magma/certs
+  - ${CONFIGS_TEMPLATES_PATH}:/etc/magma/templates
+  - ${CONFIGS_DEFAULT_VOLUME}:/etc/magma
+  - ${CONFIGS_OVERRIDE_VOLUME}:/var/opt/magma/configs
+  - ${CONFIGS_OVERRIDE_TMP_VOLUME}:/var/opt/magma/tmp
+  - ${ROOTCA_PATH}:/var/opt/magma/certs/rootCA.pem
+  - /etc/snowflake:/etc/snowflake
+  - /var/opt/magma/fluent-bit:/var/opt/magma/fluent-bit
+  - ./:/var/opt/magma/docker
+  - /var/run:/var/run
+  - /tmp:/tmp
+  - /var/log:/var/log
+  - /etc/openvswitch:/etc/openvswitch
+  - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
This is a bit ugly but I think there is no way in yaml to use and extend the list above, and I didn't want to mount the socket into all containers

Reply:
You could not use an anchor at all for the magmad list. This would reduce the complexity a bit.

Reply:
On the other hand, if it's right next to the other list, there's a better chance that someone adapting the list will notice and adapt both.